armi/physics/neutronics/latticePhysics/latticePhysicsInterface.py

```diff
@@ -228,7 +234,12 @@ def interactCoupled(self, iteration):
             This is unused since cross sections are generated on a per-cycle basis.
         """
         # always run for snapshots to account for temp effect of different flow or power statepoint
-        if self.cs[CONF_RUN_TYPE] == "Snapshots" or self.r.p.timeNode == 0:
+        secondIterOnFirstPointWithUpdatedTemps = self.r.p.timeNode == 0
```

Comment:
This setting is named `secondIterOnFirstPointWithUpdatedTemps` but it seems like it would happen on every `iteration`, not just the second one.

Reply:
yep. bad name on my part.